netci.groovy

```diff
@@ -912,7 +920,7 @@ combinedScenarios.each { scenario ->
         // Architectures.  x64 only at this point
         ['x64'].each { architecture ->
             // Put the OS's supported for coreclr cross testing here
-            Constants.crossList.each { os ->
```

Comment:
You should keep it in in crossList. You can see below this line where we filter out jobs we don't want to create (for instance, no non Checked builds for stress modes.  You can add additional conditions for OS there.